agent/eventhandler/task_handler.go

```diff
@@ -125,33 +125,47 @@ func NewTaskHandler(ctx context.Context,
 // AddStateChangeEvent queues up the state change event to be sent to ECS.
 // If the event is for a container state change, it just gets added to the
 // handler.tasksToContainerStates map.
+// If the event is for a managed agent state change, it just gets added to the
+// handler.tasksToManagedAgentStates map.
 // If the event is for task state change, it triggers the non-blocking
 // handler.submitTaskEvents method to submit the batched container state
 // changes and the task state change to ECS
 func (handler *TaskHandler) AddStateChangeEvent(change statechange.Event, client api.ECSClient) error {
 	handler.lock.Lock()
 	defer handler.lock.Unlock()
-
+	seelog.Debugf("handling Event: %v", change)
 	switch change.GetEventType() {
 	case statechange.TaskEvent:
+		seelog.Debugf("handling TaskEvent: %v", change)
```

Comment:
these could get noisy in the logs, should probably remove it. same for the one above switch statement and below ones

Reply:
removed all